src/shared/services/http-interceptors/refresh-token.interceptor.ts

```
@@ -66,8 +66,7 @@ export class RefreshTokenInterceptor implements HttpInterceptor {
 
             console.error('Failed to refresh access token', err);
 
-            // Re-throw original error
-            return throwError(error);
```

If the original error (401) is returned here, the `htt-error-interceptor` redirects to login page.  If the refired API call fails with a 422 or 500, it should return the new error so that it stays on the page. If the renewToken call falls, the api is returning 401 (oauth2.refreshRoute), which would then redirect to login page.